nix/mobile/ios/default.nix

```diff
@@ -12,6 +12,7 @@ let
   bundler-sh = callPackage ./shells/bundler.nix { };
   cocoapods-sh = callPackage ./shells/cocoapods.nix { };
   status-go-sh = callPackage ./shells/status-go.nix { inherit status-go; };
+  xcbeautify = callPackage ./xcbeautify.nix {};
```

Comment:
If I understand our structure well, this file should be in `nix/pkgs/` folder and added in `callPackage` in `nix/overlay.nix`.

Can you please have a look?